cluster/vagrant/provision-master.sh

```diff
@@ -21,16 +21,16 @@ set -e
 # See: https://github.com/mitchellh/vagrant/issues/2430
 hostnamectl set-hostname ${MASTER_NAME}
 
-# Workaround to vagrant inability to guess interface naming sequence
-# Tell system to abandon the new naming scheme and use eth* instead
-rm -f /etc/sysconfig/network-scripts/ifcfg-enp0s3
-
 # Disable network interface being managed by Network Manager (needed for Fedora 21+)
 NETWORK_CONF_PATH=/etc/sysconfig/network-scripts/
-grep -q ^NM_CONTROLLED= ${NETWORK_CONF_PATH}ifcfg-eth1 || echo 'NM_CONTROLLED=no' >> ${NETWORK_CONF_PATH}ifcfg-eth1
-sed -i 's/^#NM_CONTROLLED=.*/NM_CONTROLLED=no/' ${NETWORK_CONF_PATH}ifcfg-eth1
+for if_conf in ${NETWORK_CONF_PATH}ifcfg-*; do
+  grep -q ^NM_CONTROLLED= ${if_conf} || echo 'NM_CONTROLLED=no' >> ${if_conf}
+  sed -i 's/#^NM_CONTROLLED=.*/NM_CONTROLLED=no/' ${if_conf}
+done;
 systemctl restart network
 
+NETWORK_IF_NAME=`ls ${NETWORK_CONF_PATH}ifcfg-* | grep -v ifcfg-lo | awk -F- '{ print $3 }'`
```

Comment:
I'll add back the line that removed the enp0s3 interface. That should take
care of that particular issue.

On Thu, Jul 30, 2015 at 9:36 AM, Derek Carr <notifications@github.com>
wrote:

> In cluster/vagrant/provision-master.sh
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11469#discussion_r35884138>
> :
>
> >  systemctl restart network
> >
> > +NETWORK_IF_NAME=`ls ${NETWORK_CONF_PATH}ifcfg-* | grep -v ifcfg-lo | awk -F- '{ print $3 }'`
>
> This is not working for me on VirtualBox.
>
> I see the following result for this value:
>
> enp0s3 eth1
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11469/files#r35884138>
> .
>